DependencyInjection.Accessors/ContactsAccessor.cs

```diff
@@ -38,18 +36,13 @@ public void Update(Contact contact)
 
         public Contact Delete(Contact contact)
         {
-            //throw new NotImplementedException();
-            return Contacts.Remove(contact);
+            throw new NotImplementedException();
         }
 
         public bool Exists(int id)
         {
             return Contacts.Any(c => c.Id == id);
         }
 
-        public override int SaveChanges()
-        {
-            return SaveChanges();
```

Comment:
Is this supposed to be ```base.SaveChanges();```. Even so, it won't change the functionality.

Reply:
Yeah, probably should be. I didn't use it at all, so I have a feeling that this might actually go into an infinite loop if we don't call base. Can we add it back in though? I want them to know it's there when writing the interface